app/services/alias_updater_service.rb

```diff
@@ -7,45 +7,35 @@ def initialize(args)
   end
 
   def run
-    remove_alias_from_old_index
     assign_alias_to_new_index
-    logger.info "Alias '#{index_alias}' now pointing to '#{new_index_name}'"
+    @logger.info "Alias '#{@index_alias}' -> '#{@new_index_name}'"
   end
 
 private
 
-  attr_reader :logger, :client, :index_alias, :new_index_name
```

Comment:
Why do we need to remove the accessor methods to enable this behaviour?

Reply:
Mine and Emma's reaction when reading the code was "where do these mysterious variables come from". I think using instance variables requires less code and makes it obvious where they're coming from.

Reply:
This should be OK - me and Andrew had a chat on the phone :-)